.github/workflows/build-images.yaml

```diff
@@ -90,9 +90,6 @@ jobs:
   rust-images:
     # trigger only for push events (on protected branches as defined above) OR on PR events with the "CICD:build-images" label.
     needs: [permission-check, determine-last-green-commit]
-    strategy:
-      matrix:
-        IMAGE_TARGET: [release, test]
```

Comment:
Does unifying the rust builds of both `release` and `test` increase the worst-case build time?

Reply:
from what i can see it affects the worst case build time by a little indeed. Like 1-2 minutes or so.
But it simplifies the whole build, reduces cost etc. 
I'll keep an eye on it though, if it really turns out to be a problem we can seperate again later, but I think if we do we should probably seperate intelligently (the previous release/test seperation was for completely different reasons, not build time).